public/locales/cs/translation.json

```diff
@@ -133,6 +134,7 @@
 	"ClientFormField": {
 		"Add new input": "Přidat pole",
 		"Remove input": "Odstranit pole",
+		"Invalid format, should have minimum of 8 characters": "Nesprávný formát, musí mít minimálně 8 znaků",
```

Comment:
@Aringoaway `"Invalid format, input should have minimum of 8 characters": "Nesprávný formát, pole musí mít minimálně 8 znaků"`